app/helpers/route_overrides_helper.rb

```diff
@@ -0,0 +1,40 @@
+module RouteOverridesHelper
+  class << self
+
```

Comment:
Now that I think of it, private should not have extra space if at the start of a class.

See bbatsov/rubocop#771. If necessary, file another bug report (check against locally installed rubocop however)